md_unittest.py

```diff
@@ -0,0 +1,134 @@
+import os
+import pandas as pd
+import sqlite3
+from tempfile import TemporaryDirectory
+import unittest
+
+from md_main import check_input, insert_into_identifier, insert_into_metadata
+
+
+class TestMdMain(unittest.TestCase):
+
+    def setUp(self):
+        # Create a temporary directory for test output files
+        self.temp_dir = TemporaryDirectory()
+        self.db_connection = sqlite3.connect(':memory:')  # In-memory database
+        self.db_cursor = self.db_connection.cursor()
+
+        # Create the necessary tables for testing
+        self.db_cursor.execute('''
+            CREATE TABLE IF NOT EXISTS identifier(
```

Comment:
We should move table construction logic into .sql files, and setup a command to build/patch as needed